src/events.js

```diff
@@ -55,11 +56,17 @@ module.exports = function(ctx) {
     currentMode.mousedown(event);
   };
 
+  events.drawChanged = function(event) {
+    featuresChanged = event.features;
+  }
+
   events.mouseup = function(event) {
     mouseDownInfo.isDown = false;
     var target = getFeatureAtAndSetCursors(event, ctx);
     event.featureTarget = target;
 
+    ctx.map.fire('draw.modifiedFeature', {features: featuresChanged});
```

Comment:
Hey @mcwhittemore, so if I clear featureChanged on mousedown, it sends an empty array in mouseUp, as there is a mouseDown that comes immediately before every mouseUp. We don't have to worry about clearing it actually, because the draw.changed triggers on every single movement you make, and lines 59 to 62 ensures that featuresChanged is always storing the most up-to-date change coming from the users draw interaction.